app/models/provider.rb

```diff
@@ -26,6 +26,11 @@ class Provider < ApplicationRecord
 
   audited
 
+  scope :manageable_by, ->(provider_user) do
```

Comment:
Is this technically `with_users_manageable_by`?

Reply:
Amended here: https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/2022/commits/c76fe3e881ac35c3538a122dfc753425f8ae8eb3